enlighten/AbsorbanceFeature.py

```diff
@@ -17,10 +22,14 @@ def __init__(self,
         self.marquee        = marquee                
         self.transmission   = transmission
 
-    ## 
-    # @returns True if successfully updated ProcessedReading
-    # @todo update with numpy?
-    def process(self, processed_reading, settings):
+    def process(self, processed_reading, settings) -> bool:
```

Comment:
If you import ProcessedReading and SpectrometerSettings then you can also type annotate the parameters. I'm hoping to work towards all type annotations. Along with helping readability, I think it would be nice to have the option of using [MyPy](https://github.com/python/mypy) in a thorough manner. We could use now but since there aren't many type annotated parts I don't feel it would be very useful yet.